pkg/gen/input/ami/internal/file/ami.go

```diff
@@ -0,0 +1,35 @@
+package file
+
+import (
+	"github.com/giantswarm/devctl/pkg/gen/input"
+	"github.com/giantswarm/devctl/pkg/gen/input/ami/internal/params"
+)
+
+func NewAMIInput(p params.Params) input.Input {
+	i := input.Input{
+		Path:         params.RegenerableFileName(p, "ami.go"),
+		TemplateBody: amiTemplate,
+		TemplateData: map[string]interface{}{
+			"AMIInfoString": params.AMIInfoString(p),
+			"Package":       params.Package(p),
+		},
+	}
+
+	return i
+}
+
+var amiTemplate = `// NOTE: This file is generated by devctl. Do not edit.
+package {{ .Package }}
+
+import "encoding/json"
+
+var amiJSON = []byte({{ .AMIInfoString }})
+var amiInfo = map[string]map[string]string{}
```

Comment:
I saw the rendered file and thought this reads way better when having to debug and understand things in the repo. 

```suggestion
var amiInfo = map[string]map[string]string{}

var amiJSON = []byte({{ .AMIInfoString }})
```